Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.H

```diff
@@ -2,26 +2,58 @@
 #define WARPX_MACROSCOPICPROPERTIES_H_
 
 
+#include <AMReX_REAL.H>
+#include <AMReX_Array4.H>
+#include <AMReX_Gpu.H>
+
 #include <AMReX_MultiFab.H>
+#include <algorithm>
+#include <array>
+
+#include "Parser/WarpXParser.H"
+#include "Parser/WarpXParserWrapper.H"
 
```

Comment:
Check whether these #includes are necessary.

Reply:
Thank you. Removed them